src/app/story/storyrow/storyrow.component.css

```diff
@@ -50,11 +43,70 @@
     margin: 0;
 }
 
+.node-uid-column__display-name {
+    font-weight: bold;
+}
+
+.node-uid-column__timestamp {
+    font-size: 1.1rem;
+}
+
+.clock-icon {
```

Comment:
Could you rename this to `node-uid-column.__clock-icon` and move it to the "Elements" section?